src/modules/accommodations/components/reviews/review-card.tsx

```diff
@@ -62,9 +63,21 @@ export function ReviewCard({ review }: ReviewCardProps) {
   };
 
   const avatarContent = review.user.photo || review.user.firstName?.[0] || '';
-  const userName = `${review.user.firstName || ''} ${review.user.lastName || ''}`;
-  const userCountry = review.user.country || '';
-  const shouldShowMoreButton = review.content && review.content.length > 150;
+  const userName = `${review.user.firstName} ${review.user.lastName}`.trim();
+
+  const userCountry = review.user.country;
+  const createdAt = review.user.createdAt ? new Date(review.user.createdAt) : null;
+
+  const formattedDate = createdAt
```

Comment:
Please create util for this so that you don't have to repeat code

Reply:
Please don't use new Date